### PR TITLE
Change github_user to github_user_or_org

### PR DESCRIPTION
**Pull request by dstansby**
I didn't realise that the intent of this variable was to be the username *or* organisation where the git repository is remotely located. So when I ran the cookiecutter I put in my username, then had to change it all to the org I was actually using.

Hopefully this PR makes it clearer that this variable can be a username or an organisation.

### CODE REVIEW

**Comment by samcunliffe** [comment]
Should we stick to the GH jargon and use "owner"?

Screenshot 2024-02-06 at 14 31 24


... dunno though. It's not necessarily clearer...?

**Comment by dstansby** [comment]
Hmm, I'm 👍 to using existing terms, but I wouldn't have known off the top of my head (before reading your comment) what a GitHub owner was. Maybe worth asking a few others if they would know what "owner" meant?

**Comment by paddyroddy** [comment]
Related to #296

File: {{cookiecutter.project_slug}}/.github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ title: "[BUG]: "
 labels:
   - bug
 assignees:
-  - "{{cookiecutter.github_username}}"
+  - "{{cookiecutter.github_user_or_org}}"
```
Comment (samcunliffe) [inline {{cookiecutter.project_slug}}/.github/ISSUE_TEMPLATE/bug_report.yml:8]: Can an organisation be assigned as bug ticket?

Comment (dstansby) [reply {{cookiecutter.project_slug}}/.github/ISSUE_TEMPLATE/bug_report.yml:8]: Do you mean can an organisation the assignee on a GitHub issue? I don't think so.

**Comment by dstansby** [comment]
> Related to #296

👍 I think solving the underlying issue here (`github_user` not covering all possibilities) is best done by adding human readable prompts, so I'll close this PR and open an issue (which I should have done first anyway, sorry...)

File: {{cookiecutter.project_slug}}/.github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (samcunliffe) [reply {{cookiecutter.project_slug}}/.github/ISSUE_TEMPLATE/bug_report.yml:8]: Yes, that's what I meant. 8 am pre-coffee grammar. Sorry.

**Comment by paddyroddy** [comment]
Close branches please
image

**Pull request closed without merging**